README.md

```diff
@@ -1,15 +1,22 @@
 # Recursion Problems
 
 ## Definitions
-Define the following: 
+Define the following:
 
 - Recursion
+  - Recursion describes a method that calls itself.
 - Recursive Case
+  - A recursive case is the part of the method where recursion occurs.
 - Base Case
+  - Base case returns a value without making a recursive call. This is what recursive cases should be bringing us closer to.
 - Activation Chain/Stack
+  - An Activation chain/stack is the entire process of solving the recursive method and getting the appropriate answer.
 - Activation Record/Call
+  - An activation record/call is each little "bubble" within the activation chain. I take that to mean that an activation record occurs each time a recursive case occurs within an activation chain.
 - Infinite Recursion/Stack Overflow/Stack too deep
+  - Infinite recursion occurs when the recursive cases are not leading to a base case.
 - Tail Recursion
+  - Tail recursion occurs when the last computation within a method is a recursive call. All our recursive methods from the homework are examples of Tail recursion.
```

Comment:
None of our examples in the homework are of Tail Recursion.

I didn't actually have time to cover tail recursion in class. I may in the future, but in this moment you don't need to know it.